src/components/dev-hub/blog-post-title-area.js

```diff
@@ -19,12 +19,10 @@ const PostMetaLine = styled('div')`
     }
 `;
 
-const bulletStyling = css`
-    @media ${screenSize.xSmallAndUp} {
-        &:after {
-            content: '\u2022';
-            margin-left: ${size.tiny};
-        }
+const breakStyling = css`
+    &:after {
+        content: '|';
```

Comment:
I know this is the usual best practice, but the SEO team needs this to be in the actual text content.